core/test/unit/services/xmlrpc_spec.js

```diff
@@ -136,5 +136,41 @@ describe('XMLRPC', function () {
                 setTimeout(retry, 100);
             }());
         });
+
+        it('captures && logs XML errors from requests', function (done) {
+            var testPost = _.clone(testUtils.DataGenerator.Content.posts[2]),
+                ping1 = nock('http://rpc.pingomatic.com').post('/').reply(200,
+                `<?xml version="1.0"?>
```

Comment:
I updated this, before it was responding with `{ body: <string> }` which was incorrectly sending that objetc back as a JSON object. So now it just sends the string as with the pingomatic API